zipkin/src/main/java/com/linecorp/armeria/common/tracing/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Distributed tracing clients based on <a href="https://github.com/openzipkin/brave">Brave</a>,
+ * a Java tracing library compatible with <a href="http://zipkin.io/">Zipkin</a>.
```

Comment:
How about just:

    Common classes for distributed tracing support based on <a href="...">Brave</a>.